cfy_manager/main.py

```diff
@@ -198,10 +206,12 @@ def _finish_configuration():
 @argh.arg('--private-ip', help=PRIVATE_IP_HELP_MSG)
 @argh.arg('--public-ip', help=PUBLIC_IP_HELP_MSG)
 @argh.arg('-a', '--admin-password', help=ADMIN_PASSWORD_HELP_MSG)
+@argh.arg('--no-ssl', action='store_true', help=NO_SSL_HELP_MSG)
```

Comment:
I'm not sure we need to do this via command line. I don't like the inputs getting clogged. If we're going with SSL by default, the user who want to go the no-ssl route can edit the config file (just a preference, you can disagree, of course)